examples/rtl_433_mqtt_hass.py

```diff
@@ -560,17 +622,19 @@ def sanitize(text):
 def rtl_433_device_topic(data):
     """Return rtl_433 device topic to subscribe to for a data element"""
 
-    path_elements = []
+    #path_elements = []
```

Comment:
This would need to be removed entirely before merging. What exactly does this fix, any suggestions on how to test it?